packages/cli/src/commands/config/generate.ts

```diff
@@ -0,0 +1,123 @@
+        path: flags['schema.path'],
+        format: flags['schema.format'],
+      },
+      handler: {
+        path: flags['handler.path'],
+      },
+    };
+
+    const root = pkgDir(cwd());
+    const { config: existingConfig } = loadConfig();
+
+    const format = flags.format ?? isTypeScriptProject(flags['save-config']) ? 'ts' : 'js';
+    let configPath;
+    if (flags['save-config']) {
+      const { ext } = parse(flags['save-config']);
+      configPath = ext ? flags['save-config'] : `${flags['save-config']}.${format}`;
+    } else {
+      if (!root) {
+        this.error(
+          'Unable to find root package.json. Double-check command is being ran inside a js/ts project with a package.json',
+        );
+      }
+
+      configPath = resolve(root, `gqlmocks.config.${format}`);
+    }
+
+    if (!flags.force && existsSync(configPath)) {
+      this.error(`Bailing, file already exists at ${configPath}, use --force to overwrite`);
+    }
+
+    if (!flags['save-config']) {
+      this.warn(`All paths in the config are relative to:\n${root}\n`);
+    }
+
+    if (!config.schema.format) {
+      const defaultz = existingConfig?.schema.format || 'SDL';
+      const format = await cli.ux.prompt(`Format of GraphQL Schema? 'SDL' 'SDL_STRING' (default: ${defaultz})`, {
+        required: false,
+      });
+      config.schema.format = format || defaultz;
+    }
+
+    if (!config.schema.path) {
+      const defaultz = existingConfig?.schema?.path || 'graphql-mocks/schema.graphql';
+      const schemaPath = await cli.ux.prompt(`Path to GraphQL Schema file? (default: ${defaultz})`, {
+        required: false,
+      });
+      config.schema.path = schemaPath || defaultz;
+    }
+
+    if (!config.handler.path) {
+      const defaultz = existingConfig?.handler?.path || 'graphql-mocks/handler.ts';
+      const handlerPath = await cli.ux.prompt(`Path to GraphQL Mocks Handler file? (default: ${defaultz})`, {
+        required: false,
+      });
+      config.handler.path = handlerPath || defaultz;
+    }
+
+    if (!flags.format) {
+      this.log(
+        `ℹ️   Detected ${format} project, using ".${format}" for config. Format can be specified explicitly using the --format flag\n`,
+      );
+    }
+
+    const template = loadBlueprint(`config.${format}`);
+    const squashedConfig = { ...existingConfig, ...config };
+    const configFileContents = template(squashedConfig);
+    const tmpConfig = resolve(tmpdir(), `${randomBytes(16).toString('hex')}.${format}`);
+    writeFileSync(tmpConfig, configFileContents);
+    const { errors } = loadConfig(tmpConfig);
+
+    if (errors?.length) {
+      const formattedErrors = errors.map((error: any) => ` * ${error.message}`).join('\n');
+      this.warn(
+        `Found the follow validation errors, fix them and verify by running:\ngqlmocks config:validate\n\nValidation Errors:\n${formattedErrors}\n`,
+      );
+    }
+
+    // TODO: mkdirp path for file (same for writing config in config:generate)
```

Comment:
⚠️ TODO

Reply:
Instead of doing a `mkdir -p` for the config file it will bail and ask the user to make sure the directories exist